[Bug]: container_azure_function job in GHA failed

### Contact Details

_No response_

### What happened?

container_azure_function in .github/workflows/verify-azure-function-with-packages.yaml failed. Log is here: https://github.com/Green-Software-Foundation/carbon-aware-sdk/actions/runs/5791437176/job/15696252098

### client

WebAPI (Default)

### Relevant log output

```shell
Log of "Docker Target Final" step:


"docker rmi" requires at least 1 argument.
See 'docker rmi --help'.

Usage:  docker rmi [OPTIONS] IMAGE [IMAGE...]

Remove one or more images
Error: Process completed with exit code 1.
```
```


### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

--- a/.github/workflows/verify-azure-function-with-packages.yaml
+++ b/.github/workflows/verify-azure-function-with-packages.yaml
@@ -26,10 +26,11 @@ jobs:
         continue-on-error: true
         run: docker rm -f ${{ env.CONTAINER_RUNTIME_NAME }}
 
+      # Build and tag final image, remove dangling images only if there are any
       - name: Docker Target Final
         run: |
           docker build --no-cache . -f ${{ env.DOCKERFILE_PATH }} -t ${{ env.CONTAINER_IMAGE_NAME }}
-          docker rmi -f $(docker images -f "dangling=true" -q)
+          docker images -qf dangling=true | xargs --no-run-if-empty docker rmi
 
       - name: Docker Run Container
         run: docker run -d --name ${{ env.CONTAINER_RUNTIME_NAME }} -p 8080:80 ${{ env.CONTAINER_IMAGE_NAME }}

